src/output.c

```diff
@@ -1017,8 +1021,16 @@ void OutputRegisterRootLogger(ThreadInitFunc ThreadInit,
     TAILQ_INSERT_TAIL(&RootLoggers, logger, entries);
 }
 
+//XXX FIXME  - What the best place and names for this
```

Comment:
I think it could be moved to GlobalInits() in suricata.c. @inliniac do you confirm ?